src/pkg/index.ts

```diff
@@ -31,6 +32,12 @@ export const defaultConfig: RouteOptionsCreator = () => {
     kratosBrowserUrl: apiBaseUrl,
     faviconUrl: "favico.png",
     faviconType: "image/png",
+    isOAuthConsentRouteEnabled: () =>
```

Comment:
This still doesn't need to be a function, though. `process.env` doesn't change after start (at least it shouldn't). Or what am I missing here?

Reply:
i made it a function so we can override it :)

Reply:
but i guess it doesn't need to be per request right? :thinking: